responsibleai/requirements.txt

```diff
@@ -1,7 +1,7 @@
 dice-ml>=0.7.2,<0.8
 econml~=0.12.0
 jsonschema
-erroranalysis>=0.1.28
+erroranalysis>=0.1.29
```

Comment:
Should this be a separate PR?

Reply:
it's just updating to latest error analysis package

Reply:
Discussed over PM.